ocaml/idl/datamodel_types.ml

```diff
@@ -26,12 +26,12 @@
 
 *)
 module Date = struct
-  open Xapi_stdext_date
+  module Date = Xapi_stdext_date.Date
   include Date
```

Comment:
Would it be worth removing this `include` to only be able to access Date explicitly?

Reply:
`include Date` add the whole signature of `Date` to the current module, so I believe it's needed, unless I missed something.